ansible/logs.yml

```diff
@@ -24,13 +24,14 @@
         - subjects
       when: "'db' not in exclude_logs_from"
 
-- hosts: shared,core,invokers
+- hosts: controllers,invokers
+  serial: 1
```

Comment:
In the open source part there are neither `shared`-hosts nor `core`-hosts.
So I removed them, but added the controllers.